utils/node_local_test.py

```diff
@@ -1105,9 +1105,9 @@ def run_daos_client_cmd_pil4dfs(self, cmd, check=True, container=None, report=Tr
             # pylint: disable-next=consider-using-with
             tmp_dir = tempfile.TemporaryDirectory(prefix='pil4dfs_mount')
             cwd = tmp_dir.name
-            cmd_env['DAOS_MOUNT_POINT'] = cwd
-            cmd_env['DAOS_POOL'] = container.pool.id()
-            cmd_env['DAOS_CONTAINER'] = container.id()
+            cmd_env['D_IL_MOUNT_POINT'] = cwd
+            cmd_env['D_IL_POOL'] = container.pool.id()
+            cmd_env['D_IL_CONTAINER'] = container.id()
```

Comment:
If we want to improve testing of this mode then the lines L1131-L1134 in this file should be un-commented.  At the time the test was written they were finding files left behind in the mount point despite `pil4dfs` being used but that does appear to be fixed now.  That could be handled in a different PR though.